azure/table_azure_sql_database.go

```diff
@@ -526,14 +526,16 @@ func listSqlDatabaseVulnerabilityAssessmentScans(ctx context.Context, d *plugin.
 
 	client := sqlV5.NewDatabaseVulnerabilityAssessmentScansClientWithBaseURI(session.ResourceManagerEndpoint, subscriptionID)
 	client.Authorizer = session.Authorizer
+	var vulnerabilityAssessmentScanRecords []map[string]interface{}
 
 	op, err := client.ListByDatabase(ctx, resourceGroupName, serverName, databaseName)
 	if err != nil {
+		if strings.Contains(err.Error(), "VulnerabilityAssessmentInvalidPolicy") {
```

Comment:
@ParthaI We should add a comment, it's not easy to tell when we'd get this (hardcoded errors, strings, etc., should always have a comment so we can better understand why we need them).